src/card.css

```diff
@@ -0,0 +1,107 @@
+/* #region card */
```

Comment:
remove comments. fix all over the file

Reply:
ok) but i did it as in the lesson 
![image](https://user-images.githubusercontent.com/70288388/201165664-93ab2ba8-8569-4404-84a4-39c4f52a8a96.png)

Reply:
yes thank)  I realized that I already have separate files